packages/yoastseo/src/morphology/dutch/addNounSuffixes.js

```diff
@@ -0,0 +1,172 @@
+import { modifyStem } from "../morphoHelpers/suffixHelpers";
+import { applySuffixesToStem } from "../morphoHelpers/suffixHelpers";
+
+/**
+ * Searches for a match at the end of the stem, and returns the corresponding suffix if matched.
+ *
+ * @param {string} stemmedWord The stemmed word
+ * @param {string[]} suffixType Whether plural or diminutive suffixes are being searched for.
+ * @returns {string} The suffix.
+ *
+ */
+export function findSuffixes( stemmedWord, suffixType ) {
+	const stemEndingAndSuffix = suffixType.find( stemSuffixPair => stemmedWord.search( new RegExp( stemSuffixPair[ 0 ] ) ) !== -1 );
+
+	if ( typeof stemEndingAndSuffix !== "undefined" ) {
```

Comment:
It is more common to just check whether `stemEndingAndSuffix` is "truthy":
```suggestion
	if ( stemEndingAndSuffix ) {
```